website/screens/components/date-input/code/DateInputCodePage.tsx

```diff
@@ -190,12 +190,9 @@ const sections = [
         content: (
           <>
             <DxcText as="p">
-              Example of error handling, an error will be displayed whenever an error is sent as a
-              parameter.
-            </DxcText>
-            <DxcText as="p">
-              Take into account that it behaves differently based on the value of error (check the
-              prop above for more information).
+              The component behaviour varies depending on the value of the error prop. We recommend
```

Comment:
I suggest changing a little bit the first sentence to avoid the repetition of "prop" word: "The component behaviour varies depending on the value of `<Code>error</Code>`." 🤔